DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/igmp/test_igmp_snooping.py

```diff
@@ -0,0 +1,612 @@
+import asyncio
+import pytest
+
+from dent_os_testbed.lib.bridge.bridge_mdb import BridgeMdb
+from dent_os_testbed.lib.bridge.bridge_link import BridgeLink
+from dent_os_testbed.lib.ip.ip_link import IpLink
+
+from dent_os_testbed.utils.test_utils.tgen_utils import (
+    tgen_utils_get_dent_devices_with_tgen,
+    tgen_utils_dev_groups_from_config,
+    tgen_utils_traffic_generator_connect,
+    tgen_utils_setup_streams,
+    tgen_utils_get_traffic_stats,
+    tgen_utils_start_traffic,
+    tgen_utils_stop_traffic,
+    tgen_utils_clear_traffic_items,
+)
+
+pytestmark = [pytest.mark.suite_functional_igmp,
+              pytest.mark.asyncio,
+              pytest.mark.usefixtures("cleanup_bridges", "cleanup_tgen")]

```

Comment:
We are reusing old sessions, so `cleanup_tgen` can be omitted. When running multiple tests this will save ~20%-25% of time.
```suggestion
              pytest.mark.usefixtures("cleanup_bridges")]
```

Reply:
Done